code/modules/mob/living/slime/colours/colour_red.dm

```diff
@@ -0,0 +1,20 @@
+/decl/slime_colour/red
+	name = "red"
+	descendants = list(
+		/decl/slime_colour/red,
+		/decl/slime_colour/red,
+		/decl/slime_colour/oil,
+		/decl/slime_colour/oil
+	)
+	baby_icon =    'icons/mob/slimes/slime_baby_red.dmi'
+	adult_icon =   'icons/mob/slimes/slime_adult_red.dmi'
+	extract_icon = 'icons/mob/slimes/slime_extract_red.dmi'
+	reaction_strings = list(/decl/material/liquid/blood = "Causes all nearby slimes to enter a berserk rage.")
+
+/decl/slime_colour/red/handle_blood_reaction(var/datum/reagents/holder)
+	for(var/mob/living/slime/slime in viewers(get_turf(holder.my_atom), null))
```

Comment:
The explicit null feels confusing; this uses world.view?

Reply:
Not sure, this is copied code.

Reply:
Switched null out for a numeric range.